biogeophys/FatesHydroWTFMod.F90

```diff
@@ -1248,21 +1353,22 @@ end function dpsidth_from_th_smooth_cch
   function ftc_from_psi_smooth_cch(this,psi) result(ftc)
 
     class(wkf_type_smooth_cch) :: this
-    real(r8),intent(in) :: psi
-    real(r8)            :: ftc
-
-    real(r8)                     :: pc
-    real(r8)                     :: kr
-    real(r8)                     :: dkr_dP
-    !
-    real(r8)                     :: sat_res
-    real(r8)                     :: alpha
-    real(r8)                     :: lambda
-    real(r8)                     :: Se
-    real(r8)                     :: deltaPc
-    real(r8)                     :: dSe_dpc
-    real(r8)                     :: dkr_dSe
-
+    real(r8),intent(in)        :: psi
+    real(r8) :: ftc
+    real(r8) :: pc
+    real(r8) :: kr
+    real(r8) :: dkr_dP
+    real(r8) :: sat_res
+    real(r8) :: alpha
+    real(r8) :: lambda
+    real(r8) :: Se
+    real(r8) :: deltaPc
+    real(r8) :: dSe_dpc
+    real(r8) :: dkr_dSe
```

Comment:
Just a minor comment. My memory is a bit fuzzy on my Brooks-Corey model without looking it up; can comments on these variables be made to ease interpretability? And an overall comment citing a companion source for implementing the three main different regimes (full, smoothing, saturated).

Reply:
I created an issue here: https://github.com/NGEET/fates/issues/1193
I would just add the changes in here, but I'm not familiar with that model.